src/base/math.h

```diff
@@ -43,6 +45,15 @@ template <typename T> T sqr(const T& a) {
 	return a * a;
 }
 
+static constexpr unsigned k100PercentAsInt = 10000U;
+
+/**
+ * @brief Reads a percentage
+ * @param input A percentage in the format 12%, 12.5% or 12.53%.
+ * @return Scaled precentage as integer, where 100% corresponds to k100PercentAsInt.
```

Comment:
Ohh.... Typo `precentage`. That is unexpected, normally you are correcting my writing. ;-)